docs/user/installation.rst

```diff
@@ -89,3 +89,31 @@ Install Python modules
 .. code-block:: bash
 
   conda env create -f environment.yml
+  
+GPU Support
+===========
+
+To enable GPU support, you need to run express installation script with :code:`--gpu`. This options installs GPU-supported Tensorflow and necessary modules needed by Theano.
+
+Before you run garage, you need to specify directory CUDA library in environment varible :code:`LD_LIBRARY_PATH`. You may need to replace the directory conforming your CUDA version accrodingly. 
```

Comment:
Some small corrections:
- "... you need to specify **the** directory **for the** CUDA library in **the** environment ..."
- ".. CUDA version **accordingly**."